scraper/newsscrapper/sendData.py

```diff
@@ -0,0 +1,8 @@
+import requests
+import json
+
+url = 'http://localhost:3000/article.create'
+
+with open('spiders/articles_output.json', encoding="utf8") as f:
+   data = json.load(f)
+   requests.post(url, json = {"articles": data}, headers={'Content-Type': 'application/json'})
```

Comment:
Looks good.

Maybe add some error handling here.
the `/shared` directory has types for the API.